src/print.js

```diff
@@ -1,42 +1,179 @@
+    $removeButton.addEventListener('click', () => {
+      const indexToRemove = this.todoList.findIndex((e) => e.description === item.description);
+      if (indexToRemove > -1) {
+        this.remove(indexToRemove);
+        this.updateIndexes();
+      }
+    });
+    this.updateIndexes(); // Update the indexes after adding an item
+
+    // Add a click event listener to the edit button for the new todo item
+    // Edit button
+    const $editButton = document.getElementById(`edit-${item.description}`);
+    $editButton.addEventListener('click', () => {
+      const $inputField = document.getElementById('input'); // input div
+      $inputField.value = item.description;
+      const $addButton = document.getElementById('addBtn'); // add button
+
+      // Edit button click handler
+      $addButton.addEventListener('click', () => {
+        const newDescription = $inputField.value;
+        const indexToEdit = this.todoList.findIndex((e) => e.description === item.description);
+
+        if (newDescription !== '' && newDescription !== item.description) {
+          if (indexToEdit > -1) {
+            // Edit item
+            this.edit(indexToEdit, newDescription);
+            $editButton.id = `edit-${newDescription}`;
+            $editButton.textContent = newDescription;
+            this.updateIndexes();
+          }
+        } else {
+          // Delete item previous item
+          this.remove(indexToEdit);
+          $editButton.parentElement.removeChild($editButton);
+        }
+      });
+    });
+
+    /* const $editButton = document.getElementById(`edit-${item.description}`);
+    $editButton.addEventListener('click', () => {
+      const $inputField = document.getElementById('input'); // input div
+      $inputField.value = item.description;
+      const $addButton = document.getElementById('addBtn'); // add button
+      $addButton.addEventListener('click', () => {
+        const newDescription = $inputField.value;
+        const indexToEdit = this.todoList.findIndex((e) => e.description === item.description);
+        if (indexToEdit > -1) {
+          this.edit(indexToEdit, newDescription);
+          $editButton.id = `edit-${newDescription}`;
+          $editButton.textContent = newDescription;
+          $addButton.textContent = newDescription;
+          this.updateIndexes();
+        }
+      });
+    }); */
+
+    /*  const $editButton = document.getElementById(`edit-${item.description}`);
+    $editButton.addEventListener('click', () => {
+      const $inputField = document.getElementById('input'); // input div
+      $inputField.value = item.description;
+      $inputField.addEventListener('change', () => {
+        const newDescription = $inputField.value;
+        const indexToEdit = this.todoList.findIndex((e) => e.description === item.description);
+        if (indexToEdit > -1) {
+          this.edit(indexToEdit, newDescription);
+          /* this.updateIndexes();
```

Comment:
- [ ] Please kindly remove all forms of commented code from your codebase as it pollutes it
- [ ] Also, the edit functionality is not working. Please kindly implement it.